modules/splits/internal/key/utilities.go

```diff
@@ -23,19 +24,29 @@ func readSplitID(splitIDString string) types.ID {
 	return splitID{OwnerID: base.NewID(""), OwnableID: base.NewID("")}
 }
 
-func splitIDFromInterface(id types.ID) splitID {
-	switch value := id.(type) {
+func splitIDFromInterface(Interface interface{}) splitID {
+	switch value := Interface.(type) {
 	case splitID:
 		return value
+	case types.ID:
+		return splitIDFromInterface(readSplitID(value.String()))
 	default:
-		return splitIDFromInterface(readSplitID(id.String()))
+		panic(Interface)
 	}
 }
 
-func ReadOwnableID(assetID types.ID) types.ID {
-	return splitIDFromInterface(assetID).OwnableID
+func ReadOwnableID(id types.ID) types.ID {
+	return splitIDFromInterface(id).OwnableID
 }
 
-func ReadOwnerID(assetID types.ID) types.ID {
-	return splitIDFromInterface(assetID).OwnerID
+func ReadOwnerID(id types.ID) types.ID {
+	return splitIDFromInterface(id).OwnerID
+}
+
+func FromID(id types.ID) helpers.Key {
```

Comment:
name collides with requests' FromID and ToID.

Reply:
True but will be compunded with package name. i.e. key.FromID which will disambiguate